lawrafrontend/README.md

```diff
@@ -0,0 +1,69 @@
+# LAWRA Front-End
+
+> A chatbot that will help
+CLC clients with contacting, and receiving service from the correct legal department to improve the quality and
+efficiency of people’s interactions with CLCs.
+
+## Build Setup
+
+* Make sure you have node v12 or higher and npm/yarn installed.
+* If you're working on windows, I would recommend you use Git-bash for your Git needs and as the main terminal for this project. Make sure that you have configured Git to use `LF` as your line endings and not `CRLF` which is default on windows.
+* Make sure you cd in this directory `/lawrafrontend` and you're not in the root directory.
+
+### For yarn:
+
+``` bash
+# install dependencies
+$ yarn install
+
+# serve with hot reload at localhost:1234
+$ yarn start
+
+# build for production
+$ yarn build
+```
+
+### For npm:
```

Comment:
It is best practice to use only one package manager. Here is what yarn has to say about this
```
warning package-lock.json found. Your project contains lock files generated by tools
other than Yarn. It is advised not to mix package managers in order to avoid resolution 
inconsistencies caused by unsynchronized lock files. To clear this warning, remove
package-lock.json.
```

Reply:
@thaingu